src/components/Tooltip/tests/Tooltip.test.tsx

```diff
@@ -64,4 +66,21 @@ describe('<Tooltip />', () => {
     wrapperComponent.simulate('mouseLeave');
     expect(findByTestID(tooltip, 'TooltipOverlayLabel').exists()).toBe(false);
   });
+
+  it('closes itself when escapse is press on keyup', () => {
+    const tooltip = mountWithApp(
+      <Tooltip active content="This order has shipping labels.">
+        <div>Order #1001</div>
+      </Tooltip>,
+    );
+
+    (tooltip.children[0] as {
+      trigger(val: 'onKeyUp', values: {keyCode: Key}): void;
+    }).trigger('onKeyUp', {
+      keyCode: Key.Escape,
+    });
```

Comment:
We can avoid this casting by just finding the thing you need directly. IMO that's the same fragility as saying children[0]. 

```suggestion
    tooltip.find('span')!.trigger('onKeyUp', {
      keyCode: Key.Escape,
    });
```